planetary-social/nos
# Update profile cards for the Discover tab (including search results)

#1004
#1005
#1006 

## Description
Updates the design of the profile cards for the Discover tab. Currently only visible in search results and the `@mention` search.

## How to test
1. Tap Discover
2. Search for a name like "Nos"
3. Observe new design in search results
4. Follow and unfollow some accounts
5. Tap Post
6. Type "@"
7. Observe new design in mentions search

## Screenshots
| Before | After |
|--------|--------|
| ![Simulator Screenshot - iPhone 15 Pro Max - 2024-04-17 at 16 36 38](https://github.com/planetary-social/nos/assets/59564/0ba69b9d-29b4-4a67-b156-87d54cec45cf) | ![Simulator Screenshot - iPhone 15 Pro Max - 2024-04-17 at 16 35 25](https://github.com/planetary-social/nos/assets/59564/302cdb30-fd04-4c3e-aa67-3b3d8fe14ee4) |
| ![Simulator Screenshot - iPhone 15 Pro Max - 2024-04-17 at 16 37 38](https://github.com/planetary-social/nos/assets/59564/077b1ccb-c417-4a18-a98f-23dba1b2dd7e) | ![Simulator Screenshot - iPhone 15 Pro Max - 2024-04-17 at 16 39 36](https://github.com/planetary-social/nos/assets/59564/6063a629-e306-437c-aacb-afc7e56f0a37) |

## Comments
Now that I've been using this for a short amount of time, I wonder if it was a bad idea to remove the user's Name from the card. Seems like in some cases, their NIP-05 username isn't very good, or very descriptive, and their Name is better and easier to distinguish.

In the ticket, there's an unresolved [comment](https://github.com/planetary-social/nos/issues/1004#issuecomment-2061623478) about switching from "Muted" to an icon in the card. Until that's resolved, we can't close the ticket and this PR will continue to use "Muted" as text rather than an icon.
